.bazelrc

```diff
@@ -6,4 +6,8 @@ build:windows --compiler="clang-cl" --cxxopt="/std:c++17"
 
 build:macos --cxxopt=-std=c++17
 
-build:linux --cxxopt=-std=c++17
\ No newline at end of file
+build:linux --config=clang_config --cxxopt=-std=c++17
+
+build:clang_config --crosstool_top=//toolchain:clang_suite
```

Comment:
And let's name the toolchain appropriately too:

`build:linux_usr_lib_llvm_10_config --crosstool_top=//toolchain:linux_usr_lib_llvm_10_toolchain`